documentation/API-design-guidelines.md

```diff
@@ -1353,20 +1364,22 @@ curl -X 'POST' \
  ```
  ```json 
 {
-  "eventSubscriptionId": "456g899g",
-  "event": {
-    "eventType": "ROAMING_STATUS",
-    "eventTime": "2023-01-19T13:18:23.682Z",
-    "eventDetail": {
-      "ueId": {
-        "ipv4Addr": "192.168.0.1"
-      },
-      "uePort": 5060,
-      "roaming": true,
-      "countryCode": 208,
-      "countryName": "FR"
-    }
-  }
+  "id": 123654,
+  "source": "supertelco.notificationSendServer12",
```

Comment:
`source` MUST be a non-empty URI-reference
" The application MAY use UUIDs, URNs, DNS authorities or an application-specific scheme to create unique `source `identifiers."

```suggestion
"source": "https://notificationSendServer12.supertelco.com",
```